oggm/utils/_workflow.py

```diff
@@ -2546,7 +2559,16 @@ def write_monthly_climate_file(self, time, prcp, temp,
 
             v = nc.createVariable('prcp', 'f4', ('time',), zlib=zlib)
             v.units = 'kg m-2'
-            v.long_name = 'total monthly precipitation amount'
+
+            # check if prcp has really monthly format 
+            if len(prcp)==(nc.hydro_yr_1-nc.hydro_yr_0+1)*12:
+                v.long_name = 'total monthly precipitation amount'
+            else:
+                v.long_name = 'total monthly precipitation amount'
+                import warnings
+                warnings.warn("there might be a conflict in the prcp"
+                              "timeseries, please check!")
```

Comment:
Why is this needed?

Reply:
ok, maybe this won't happen anymore, I just thought that if somehow someone uses daily instead of monthly data as input, they would get a warning at least, but I won't do this anymore as I have now a separate function, so  I will remove it